nilearn/decomposition/canica.py

```diff
@@ -127,13 +127,18 @@ def __init__(self, mask=None, n_components=20, smoothing_fwhm=6,
             mask_strategy=mask_strategy, mask_args=mask_args,
             memory=memory, memory_level=memory_level,
             n_jobs=n_jobs, verbose=verbose)
+
+        if(threshold > n_components):
+            raise ValueError("Threshold must not be higher than number"
```

Comment:
In the error message, there is no space after number. You can leave space after number before ".
Error message can be seen here, https://travis-ci.org/nilearn/nilearn/jobs/218779100#L2729